src/components/modals/LibraryCardFormModal.jsx

```diff
@@ -3,18 +3,16 @@ import {cloneDeep, includes, isEmpty, isNil, isObject} from 'lodash/fp';
 import {Styles, Theme} from '../../libs/theme';
 import CloseIconComponent from '../CloseIconComponent';
 import Modal from 'react-modal';
-import {SearchSelect} from '../SearchSelect';
 import Creatable from 'react-select/creatable';
 import SimpleButton from '../SimpleButton';
 import {LibraryCardAgreementTermsDownload} from '../LibraryCardAgreementTermsDownload';
 
 const FormFieldRow = (props) => {
-  const { card, dropdownOptions, updateInstitution, updateUser, modalType, setCard } = props;
+  const { card, dropdownOptions, updateUser, modalType, setCard } = props;
 
   const cardlessOptions = dropdownOptions.filter((option) => {
     const libraryCards = option.libraryCards || [];
-    const savedCard = libraryCards.find(({institutionId}) => institutionId === card.institutionId);
-    return isNil(savedCard);
+    return isEmpty(libraryCards);
```

Comment:
I think this can be simplified as `_.isEmpty(undefined)` is `true`, so `|| []` is redundant
```suggestion
  const cardlessOptions = dropdownOptions.filter((option) => isEmpty(option.libraryCards));
```